frontend/package.json

```diff
@@ -5,7 +5,7 @@
   "scripts": {
     "build": "rollup -c",
     "dev": "rollup -c -w",
-    "start": "sirv public"
+    "start": "sirv public --single --port 5555"
```

Comment:
Am wondering why this change in port is required. Not too familiar with `sirv public` command.

Reply:
It wasn't working for me with the default port(only the home page) when running only the frontend, so adding those attributes fixed it